airtime_mvc/application/forms/LiveStreamingPreferences.php

```diff
@@ -125,12 +138,12 @@ public function updateVariables()
                 array ('ViewScript',
                     array (
                       'viewScript'                  => 'form/preferences_livestream.phtml',
-                      'master_source_host'          => isset($masterSourceParams["host"])?$masterSourceParams["host"]:"",
-                      'master_source_port'          => isset($masterSourceParams["port"])?$masterSourceParams["port"]:"",
-                      'master_source_mount'         => isset($masterSourceParams["path"])?$masterSourceParams["path"]:"",
-                      'show_source_host'            => isset($showSourceParams["host"])?$showSourceParams["host"]:"",
-                      'show_source_port'            => isset($showSourceParams["port"])?$showSourceParams["port"]:"",
-                      'show_source_mount'           => isset($showSourceParams["path"])?$showSourceParams["path"]:"",
+                      'master_source_host'          => isset($masterSourceHost)?$masterSourceParams["host"]:"",
+                      'master_source_port'          => isset($masterSourcePort)?$masterSourceParams["port"]:"",
+                      'master_source_mount'         => isset($masterSourceMount)?$masterSourceParams["path"]:"",
+                      'show_source_host'            => isset($showSourceHost)?$showSourceParams["host"]:"",
+                      'show_source_port'            => isset($showSourcePort)?$showSourceParams["port"]:"",
+                      'show_source_mount'           => isset($showSourceMount)?$showSourceParams["path"]:"",
                       'isDemo'                      => $isDemo,
                     )
```

Comment:
I wasn't sure if this was important. The masterSourceParams was an array that never got filled that came from the GetMasterConnectionUrl function (but that function seemed to return a URL), so I wasn't sure how the key value array was supposed to be built for the airtime.pro functionality of the form.